src/components/TodoMenu.js

```diff
@@ -0,0 +1,32 @@
+import listTodos from "../utils/listTodos.js"
+
+export default function TodoMenu({initialState, onSelectMenu}) {
+    const $all = document.querySelector('.all')
+    const $active = document.querySelector('.active')
+    const $completed = document.querySelector('.completed')
+   
+    this.state = initialState
+
+    this.setState = (newState) => {
+        this.state = newState
+    }
+
+    const addClickEvent = ($element) => {
+        $element.addEventListener('click', (e) => {
+            const $filter = e.target.closest('a')
+
+            const $prevMenu = document.querySelector(`a[class*="${this.state}"]`)
+            $prevMenu.classList.remove('selected')
+
+            $filter.classList.add('selected')
+
+            onSelectMenu($filter.classList[0])
```

Comment:
```suggestion
            onSelectMenu($filter.className);
            $filter.classList.add("selected");
```
`selected` 클래스명을 붙이기 전에는 클래스 이름이 `all`, `active`, `completed` 중 하나니까 `onSelectMenu`를 먼저 하는 방식으로 바꿔보았습니다!
더 명시적으로 수정하는게 좋을지 조언해주시면 감사하겠습니닷